packages/main/src/plugin/index.ts

```diff
@@ -1495,6 +1496,13 @@ export class PluginSystem {
       },
     );
 
+    this.ipcHandle(
+      'image-registry:searchImages',
+      async (_listener, registryName: string, term: string, limit: number): Promise<ImageSearchResult[]> => {
+        return imageRegistry.searchImages(registryName, term, limit);
```

Comment:
I think this file is requiring a change (probably the file has not been committed with the change ? to use the options parameter rather than several parameters)

Reply:
Yes right, I just pushed a new commit for this. Can you please validate the change?